spec/support/shared/shopify_shop.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.shared_context 'shopify_shop' do
+  let(:refund_amount) { 100 }
+  let(:shopify_payment_method) do
+    gateway = Spree::Gateway::ShopifyGateway.new
+    gateway.set_preference(:api_key, ENV.fetch('SHOPIFY_API_KEY'))
+    gateway.set_preference(:password, ENV.fetch('SHOPIFY_PASSWORD'))
+    gateway.set_preference(:shop_name, ENV.fetch('SHOPIFY_SHOP_NAME'))
+    gateway
+  end
+
+  def mock_request(endpoint, extension)
```

Comment:
If you get a more complete backtrace, does that reveal anything interesting? I've seen errors like that when I was trying to retrieve a Transaction, because the correct syntax is `Transaction.find(id, params: { order_id: order_id })`. Otherwise, AResource didn't know how to build the full URL.